bolt-design-system/bolt
# Docs | Fix | Fix button docs + address several docs site  bugs

## Jira
N/A

## Summary
Fixed duplicated button doc pages, updated related Nightwatch tests, and addressed two minor docs site related issues involving the docs site specific grid + code blocks being unable to scroll vertically.

## Details
- Resolving merge conflicts from in several concurrent updates led to some button docs pages being duplicated / not displaying as expected.
- Reorganizes the button component demos that involve server-side rendering to declutter the main docs overview + update Nightwatch tests that check to make sure that SSR functionality is working as expected.
- This PR also addresses three other docs site-related issues identified when testing: 
    1. Fixed an issue with content sometimes overflowing from the custom CSS grid layout used on the docs site
    2. Fixed a separate layout issue with shorter docs pages not filling up the available height
    3. Added a small update to the Bolt Global styles handling raw code / pre tags. These HTML elements can now scroll horizontalally _and now vertically_ if the content exceeds the elements maximum size

## How to test
Run the branch locally and check button docs. Make sure there are no duplicates for demonstrating the same thing.
